store/src/java/com/zimbra/cs/zimlet/ZimletUtil.java

```diff
@@ -673,7 +673,20 @@ public static void deployZimletLocally(ZimletFile zf) throws IOException, Zimlet
 
         // install the config
         if (zf.hasZimletConfig()) {
-            installConfig(zf.getZimletConfig());
+            ZimletConfig zc = getZimletConfig(zimletName);
+            if (zc != null) {
+                Version zimletConfigVersion = new Version (zf.getZimletConfig().getVersion().toString());
+                Version zimletConfigVersionInLDAP = new Version (zc.getVersion().toString());
+                if (zimletConfigVersion.compareTo(zimletConfigVersionInLDAP)> 0) {
+                    installConfig(zf.getZimletConfig());
+                }
+                if (zimletConfigVersion.compareTo(zimletConfigVersionInLDAP) == 0) {
+                    return;
```

Comment:
isn't this going to skip rest of the processing? don't we need to just skip calling installConfig?

Reply:
yes this will skip rest of the processing, updated the code for same.